src/conic.jl

```diff
@@ -844,17 +844,29 @@ function MathProgBase.optimize!(m::PajaritoConicModel)
     cputime_conic += time() - start_conic
 
     ini_conic_status = MathProgBase.status(ini_conic_model)
-    ini_conic_objval = MathProgBase.getobjval(ini_conic_model)
     if ini_conic_status == :Optimal || ini_conic_status == :Suboptimal
         separator = MathProgBase.getsolution(ini_conic_model)
         separator = addSlackValues(m, separator)
         @assert length(separator) == m.numVar
         addPrimalCuttingPlanes!(m, mip_model, separator, [], zeros(m.numVar))
-    else
-        (m.verbose > 1) && println("Conic Relaxation Infeasible")
+    elseif ini_conic_status == :Infeasible
+        (m.verbose > 1) && println("Conic Relaxation Infeasible.")
         m.status = :Infeasible
-        return       
+        m.solution = fill(NaN, m.numVar)
+        return    
+    # TODO Figure out the details for this condition to hold!   
+    elseif ini_conic_status == :Unbounded
+        (m.verbose > 1) && println("Conic Relaxation Unbounded.")
```

Comment:
I would turn this into a ``warn`` because the ``:InfeasibleOrUnbounded`` status is confusing